packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js

```diff
@@ -1,35 +1,21 @@
-import React, { useReducer, useState, useRef, useEffect } from 'react';
-import { includes, isEmpty, toString, isEqual, intersectionWith } from 'lodash';
+import React, { useReducer, useState, useEffect } from 'react';
+import { includes, toString, isEqual, intersectionWith } from 'lodash';
 import { useHistory, useLocation } from 'react-router-dom';
+import { useIsMounted } from '@buffetjs/hooks';
```

Comment:
There's a bug with this hook can you use the previous logic and add a comment so we can migrate it later when the bug has been fixed?